docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -0,0 +1,594 @@
+
+Apply your update:
+
+```shell
+kubectl apply -f app.yaml
+```
+
+After about 30 seconds you should now see two `keptnappversions`:
+
+```shell
+kubectl -n keptndemo get keptnappversion
+```
+
+Expected output:
+
+```shell
+NAME                          APPNAME        VERSION   PHASE
+keptndemoapp-0.0.1-***  keptndemoapp   0.0.1     Completed
+keptndemoapp-0.0.2-***  keptndemoapp   0.0.2     AppDeploy
+```
+
+Wait until the `PHASE` of `keptndemoapp-0.0.2` is `Completed`.
+This signals that the deployment was successful and the pod is running.
+
+View the Keptn Applications Dashboard and you should see the DORA metrics and an OpenTelemetry trace:
+
+![keptn applications dashboard](assets/keptnapplications.png)
+
+![deployment trace](assets/deploymenttrace.png)
+
+## Step 14: More control over KeptnApp
+
+To customize workloads and checks associated with the application, we can edit the autogenerated KeptnApp or create our own.
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnApp
+metadata:
+  name: <app-name>
+  namespace: <app-namespace>
+spec:
+  version: "x.y"
+  revision: x
+  workloads:
+  - name: <workload1-name>
+    version: <version-string>
+  - name: <workload2-name>
+    version: <version-string>
+  preDeploymentTasks:
+  - <list of tasks>
+  postDeploymentTasks:
+  - <list of tasks>
+  preDeploymentEvaluations:
+  - <list of evaluations>
+  postDeploymentEvaluations:
+  - <list of evaluations>
+```
+
+## Fields
+
+- **apiVersion** -- API version being used.
+- **kind** -- Resource type.
+   Must be set to `KeptnApp`
+
+- **metadata**
+  - **name** -- Unique name of this application.
+    Names must comply with the
+    [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
+    specification.
+
+- **spec**
+  - **version** -- version of the Keptn application.
+    Changing this version number causes a new execution
+    of all application-level checks
+  - **revision** -- revision of a `version`.
+    The value is an integer that can be modified
+    to trigger another deployment of a `KeptnApp` of the same version.
+    For example, increment this number to restart a `KeptnApp` version
+    that failed to deploy, perhaps because a
+    `preDeploymentEvaluation` or `preDeploymentTask` failed.
+    See
+    [Restart an Application Deployment](../../implementing/restart-application-deployment/)
+    for a longer discussion of this.
+  - **workloads**
+    - **name** - name of this Kubernetes
+      [workload](https://kubernetes.io/docs/concepts/workloads/).
+      Use the same naming rules listed above for the application name.
+      Provide one entry for each workload
+      associated with this Keptn application.
+    - **version** -- version number for this workload.
+      Changing this number causes a new execution
+      of checks for this workload only,
+      not the entire application.
```

Comment:
This isn't completely true. I would just leave the info about that it represents the version of the workload and if the workload (and it's version) was updated, this needs to be updated as well